packages/govern-create/helpers/logger.ts

```diff
@@ -0,0 +1,46 @@
+import chalk, { ForegroundColor } from 'chalk'
```

Comment:
I'm against using an additional dependency just to define a color in such an important place. Just pass the colors as documented. Any additional dependency means an additional security risk.